### PR TITLE
[5.0] STMP restart for Swift_Mailer

### PR DESCRIPTION
**Pull request by YOzaz**
Second try.

### CODE REVIEW

File: src/Illuminate/Mail/Mailer.php [hunk 1]
```diff
@@ -335,6 +337,44 @@ protected function parseView($view)
...
 	/**
+	 * Reset Swift Mailer SMTP transport adapter
+	 *
+	 * @return void
+	 */
+	protected function resetSwiftTransport()
+	{
+		$transport = $this->getSwiftMailerTransport();
+
+		if ( ! isset($transport))
```
Comment (GrahamCampbell) [inline src/Illuminate/Mail/Mailer.php]: One line this please. Also, isset is wrong here.

**Comment by taylorotwell** [comment]
I don't want to force this on everyone. Not sure if it will break some existing applications using SMTP.

**Comment by lukepolo** [comment]
Can we at least document this somewhere? It would save a lot of people time researching this only to see  that Laravel does not retry the connection if your in the daemon queue (knowing that it makes sense) but seems to be a frequent occurrence.

**Comment by YOzaz** [comment]
@lukepolo suppose you can make a pull request for docs here: https://github.com/laravel/docs/blob/5.2/queues.md#coding-considerations-for-daemon-queue-listeners

**Comment by YOzaz** [comment]
@lukepolo https://github.com/laravel/framework/commit/af8eb1face000f82e5c85e6eb822075fc313cbb9

**Pull request closed without merging**